### PR TITLE
Add: Discord Bot settings for onboarding

### LINKED ISSUE #345: [Docs]: add intents, permissions, and scopes to contributing docs
### aspect

Contributing

### Suggestion?

title

### Possible Solutions

_No response_

### Extra Information

_No response_

### PR DESCRIPTION
**Pull request by uerk-io**
# Description

Adds on-boarding for new users setting up their development Discord Bot

Fixes #345

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -113,6 +113,54 @@ yarn lint:fix
 yarn build
 ```
 
+## Setting up your Developer Bot
+
+Head to the [Discord Developer Portal](https://discord.com/developers/applications), and create a new application (call it something like "AO Dev Test"), and optionally a testing server in Discord.
+
+In the portal, click "Bot" and add a bot. You'll need to enable Server Members Intent, and Message Content Intent like so:
```
Comment (RhysSullivan) [inline CONTRIBUTING.md:120]: Make this in the following format:

Enable the following intents:
- Server members
- Message content

It makes it easier to skim

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -113,6 +113,54 @@ yarn lint:fix
...
+
+<div align="center">
+  <figure>
+    <img src="./assets/OAuth2-Settings.png" alt="OAuth2 Settings"/>
+  </figure>
+</div>
+
+You can then generate a link from `OAuth2->URL Generator` using the `bot` and `applications.commands`, with the same permissions.
+
+To make this quicker for you, you can use: `https://discord.com/oauth2/authorize?client_id=YOUR_ID&permissions=328565083201&scope=bot+applications.commands`, which has the correct permissions integer set. Just fill in your client ID, and you'll be ready to go!
```
Comment (RhysSullivan) [inline CONTRIBUTING.md:160]: Maybe lets link to https://discordapi.com/permissions.html here?

**Review by RhysSullivan — CHANGES_REQUESTED** [review]
Looks good, two minor tweaks requested

File: CONTRIBUTING.md (hunk 2 above)
Comment (uerk-io) [reply CONTRIBUTING.md:160]: Add to existing content, or replace 158-160?

Comment (RhysSullivan) [reply CONTRIBUTING.md:160]: Lets add to existing content, say "replace your id in this link or use this generator"

**Review by RhysSullivan — APPROVED** [review]

**Pull request merged**